go-controller/pkg/ovn/egressip.go

```diff
@@ -1981,8 +2011,12 @@ func (e *EgressIPController) initClusterEgressPolicies(nodes []interface{}) erro
 	if err != nil {
 		return fmt.Errorf("failed to get default network: %v", err)
 	}
+	localNodeName, err := e.getALocalZoneNodeName()
+	if err != nil {
+		klog.Warningf(err.Error())
```

Comment:
Its intentional because we may not require this info if we do not process L2 networks and if we need to process a L2 network later on in this function, an error is returned.